docs/developer-guide/ceps/proposed/cep-003-remove-image-parameters-in-camera-frame.rst

```diff
@@ -0,0 +1,52 @@
+.. _cep-003:
+
+
+************************************************************
+CEP 3 - Dropping support for image parameters in CameraFrame
+************************************************************
+
+* Status: draft
+* Discussion: NA
+* Date accepted: NA
+* Last revised: 2023-09-22
+* Author: Maximilian Linhoff
+* Created: 2023-09-22
+
+Abstract
+========
+
+Currently, ctapipe supports computing all image parameters in two variants:
+
+* Using a ``CameraGeometry`` where pixel coordinates are expressed in ``CameraFrame``, i.e.
+  in length units (most commonly meters) on the camera focal plane.
+* Using a ``CameraGeometry`` where pixel coordinates are expressed in ``TelescopeFrame``, i.e.
+  in angular units (most commonly degree) on sky.
+
+We propose to drop support for the first, to simplify code in multiple places and reduce
+possibility for confusing the two similar variants of the image parameters.
+
+The overhead of supporting both ``TelescopeFrame`` and ``CameraFrame`` representations
+of the image parameters is quite significant, as it e.g. requires dealing with both
+possible definitions in all Hillas-style dl2 reconstructors.
+
+Advantages of computation in TelescopeFrame
+===========================================
+
+Computing the image parameters in ``TelescopeFrame`` – angular units on the sky –
+has the following advantages:
+
+* Parameters are easier to compare across different telescope types.
+* Pointing corrections can directly be applied in the conversion from ``CameraFrame``
+  to ``TelescopeFrame`` and are then automatically included in the image parameters,
+  which is much more straight forward than trying to correct image parameters that
+  are affected to different degrees after they have been computed.
+* Conversion from ``CameraFrame`` to ``TelescopeFrame`` will include any necessary
+  special handling of the curved cameras of dual mirror telescopes.
+
```

Comment:
I would add here that the conversion from CameraFrame to TelescopeFrame can also include pointing corrections for offset and rotation that need to be applied before parameterization.

Reply:
Sorry, somehow I missed it in point 1!